opencog/CMakeLists.txt

```diff
@@ -22,6 +23,11 @@ ADD_SUBDIRECTORY (utils)
 IF (HAVE_CYTHON)
 	ADD_SUBDIRECTORY (cython)
 ENDIF (HAVE_CYTHON)
+
+IF (HAVE_ATOMSPACE AND HAVE_GUILE)
+	ADD_SUBDIRECTORY (reducture)
+ENDIF (HAVE_ATOMSPACE AND HAVE_GUILE)
```

Comment:
Now that the URE lives in its own repository you should check for its presence, see https://github.com/opencog/pln/blob/master/CMakeLists.txt for examples how that's being done.

Reply:
Also, I think `urereduct` (or `ure-reduct`) would be a better name, just to follow English naming convention.